embeddings/pipeline/pipelines_metadata.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Dict, Literal, Optional, Tuple, TypedDict, TypeVar
+
+from embeddings.data.io import T_path
+
+
+class PathMetadata(TypedDict, total=False):
+    output_path: T_path
```

Comment:
It's just for type-safe unwrapping parameters of given Pipeline. 
In concrete example:
```
pipeline = HuggingFaceSequenceLabelingPipeline(**metadata)
```
Where metadata now can be type-safe Dict instead of dict that can contain everything.